src/base/bittorrent/session.cpp

```diff
@@ -417,6 +417,13 @@ Session::Session(QObject *parent)
     , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), MixedModeAlgorithm::TCP
         , clampValue(MixedModeAlgorithm::TCP, MixedModeAlgorithm::Proportional))
     , m_multiConnectionsPerIpEnabled(BITTORRENT_SESSION_KEY("MultiConnectionsPerIp"), false)
+    , m_validateHTTPSTrackerCertificate(BITTORRENT_SESSION_KEY("ValidateHTTPSTrackerCertificate"), false)
+    , m_httpsTrackerTLSVersion(BITTORRENT_SESSION_KEY("HTTPSTrackerTLSVersion"), TLSVersion::TLS12
+#if (TORRENT_USE_TLS13)
```

Comment:
The macro is an int so we need to compare it explicitly:
`#if (TORRENT_USE_TLS13 == 1)`

Reply:
Done

Reply:
the previous code would work too. (it would not have worked if it used `#ifdef` however)